app/graphql/enums/filter_match_enum.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Enums::FilterMatchEnum < Types::BaseEnum
+  value 'ABSENT', 'absent (negated present)', value: 'absent'
+  value 'CONTAINS', 'contains', value: 'contains'
+  value 'MATCHES', 'matches', value: 'matches'
+  value 'OMITS', 'omits (negated contains)', value: 'omits'
+  value 'PRESENT', 'present', value: 'present'
+  value 'VARIES', 'varies (negated matches)', value: 'varies'
```

Comment:
In Hyacinth 2, we call this "DOES_NOT_EQUAL".  Can we keep that term?  I feel like "varies" is a confusing term to pick in the custom filter selector (e.g. "original file path > varies > carnegie")

Reply:
I struggled with this in my head, since this enum is kind of a mish-mash of verbs, adjectives and phrases 😟 I don't know that the user-facing labels have to be the same as the enum values, but sure - especially if we use 'equals' instead of 'matches'.

Reply:
so `EQUALS/DOES_NOT_EQUAL`, `CONTAINS/DOES_NOT_CONTAIN`, `STARTS_WITH/DOES_NOT_START_WITH`, `HAS_A_VALUE/DOES_NOT_HAVE_A_VALUE`?

Reply:
Thanks!  That sounds good!  DOES_NOT_START_WITH isn't required, but if it's easy to add I'm good with having it in there too.

I could live with `HAS_A_VALUE/DOES_NOT_HAVE_A_VALUE`, but I have a slight preference for `IS_PRESENT/IS_ABSENT` because it might be less error prone for API users:

"collection > has a value > [am I allowed to match a value here?]"
vs.
"collection > is present > [hopefully wouldn't make as much sense to put anything after 'is present'?]"

In the end though, it's probably just personal preference and being used to the HYC2 "present" vs. "absent" terms.

Reply:
`PRESENTS/DOES_NOT_PRESENT` 😬

Reply:
Mostly I just want it to be clear what the negated matches are, and what they're negating.

Reply:
AHH! Anything but `PRESENTS/DOES_NOT_PRESENT`!  Haha.

👍  for negated match clarity.

Maybe `IS_PRESENT/IS_NOT_PRESENT`?